code/modules/mob/living/simple_animal/constructs/constructs.dm

```diff
@@ -233,6 +237,7 @@
 	resistance = 10
 	var/energy = 0
 	var/max_energy = 1000
+	can_climb = 1 //It flies, its magic
```

Comment:
I feel juggernauts and artificers should not be allowed to climb, mostly because they are will also fall down in holes, while wraiths and harversts seems to be far more mobile than the others.